### DISCUSSION TITLE
How we can create multilangual documentation.

### DISCUSSION THREAD

**Discussion by otabekoff** [nuxt-content/docus#783] [Q&A]
I am asking about:

- Translating pages with any layout(default, page & etc).
- Translating content specific texts (like next-prev buttons).
- Creating translation of text with variables and using them in anywhere.
- And more...

Is this possible or being implemented or will be implemented?

**Comment by lucpotage** [comment]
Hi @otabekoff,

There is a related issue in Nuxt Content repo. It should help: https://github.com/nuxt-modules/i18n/discussions/1872

I suppose you could create different lang directories in both `content/` and `public/ ` root directories:

```
/content
  /en
    /about.md
  /fr
    /about.md
/public
  /en
    /image.png
  /fr
    /image.png
```
